content/docs/setup/kubernetes/upgrade/notice/index.md

```diff
@@ -1,36 +1,51 @@
+reliability in a variety of Kubernetes environments.
 
-- Outbound traffic policy now defaults to `ALLOW_ANY`.  Traffic to unknown ports will be forwarded as-is. Traffic to known ports (e.g., port 80) will be matched with one of the services in the system and forwarded accordingly.
-- During sidecar routing to a service, destination rules for the target service in the same namespace as the sidecar will take precedence, followed by destination rules in the service’s namespace, and finally followed by destination rules in other namespaces if applicable.
-- We recommend storing gateway resources in the same namespace as the gateway workload (e.g., `istio-system` in case of `istio-ingressgateway`).  When referring to gateway resources in virtual services, use the namespace/name format instead of using `name.namespace.svc.cluster.local`.
-- The optional egress gateway is now disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles by default.  If you need to control and secure your outbound traffic through the egress gateway, you will need to enable `gateways.istio-egressgateway.enabled=true` manually in any of the non-demo profiles.
+The following CRDs remain:
 
-## Policy & Telemetry
+| Custom Resource Definition name | Purpose |
+| --- | --- |
+| `adapter`| Specification of Istio extension declarations |
+| `attributemanifest` | Specification of Istio extension declarations |
+| `template` | Specification of Istio extension declarations |
+| `handler` | Specification of extension invocations |
+| `rule` | Specification of extension invocations |
+| `instance` | Specification of extension invocations |
 
-- `istio-policy` check is now disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles.  This change is only for `istio-policy` and not for `istio-telemetry`.  In order to re-enable policy checking, run `helm template` with `--set global.disablePolicyChecks=false` and re-apply the configuration.
-- The Service Graph component has now been deprecated in favor of [Kiali](https://www.kiali.io/).
+In the event you are using the removed mixer configuration schemas, set
+the following Helm flags during upgrade of the main Helm chart:
+`--set mixer.templates.useTemplateCRDs=true --set mixer.adapters.useAdapterCRDs=true`
 
 ## Security
 
-- RBAC configuration has been modified to implement cluster scoping.  The `RbacConfig` resource has been replaced with the `ClusterRbacConfig` resource. Refer to [Migrating `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrade/steps/#migrating-from-rbacconfig-to-clusterrbacconfig) for migration instructions.
+{{< tip >}}
+Istio 1.0's `RbacConfig` was deprecated in Istio 1.1.  It has been replaced
+with a new custom resource `ClusterRbacConfig` in Istio 1.2.0.
+{{< /tip >}}
+
+- In Istio 1.2 the `RbacConfig` custom resource is replaced by the `ClusterRbacConfig`
```

Comment:
Same here.

Reply:
why are we documenting 1.1 changes in the 1.2 upgrade notes?

Reply:
should be removed :)

Reply:
We still want to describe how customers can upgrade from 1.0 to 1.2.